### ISSUE TITLE
More architectures in node.js lib

### ISSUE THREAD

**Issue by Coffer-studio** [alphacep/vosk-api#620]
Error with run node.js lib on pi4:
vosk/lib/linux-x86_64/libvosk.so: wrong ELF class: ELFCLASS64

Are there any solutions?

**Comment by Coffer-studio** [comment]
I just copied this the file from python library. And it's works

**Comment by sadrasabouri** [comment]
You mean you do not have the problem now. right?
So would you like to close this issue?

**Comment by nshmyrev** [comment]
> So would you like to close this issue?

Thank you. It is ok to keep the issue open. We want to add binary for RPI into node package one day.

**Comment by nshmyrev** [comment]
It would be also nice to add Windows 32-bit

**Comment by nshmyrev** [comment]
See #755

**Comment by akunerio** [comment]
> I just copied this the file from python library. And it's works

I have similar issue and its solved. 
The issue was found when I run the demo in nodejs folder, because ffi-napi
`Error: ENOENT: no such file or directory, open '/home/Downloads/vosk-api/nodejs/lib/linux-x86_64/libvosk.so'`

Then, I just copy the libvosk.so from python module vosk from '/usr/local/lib/python3.9/dist-packages/vosk/libvosk.so'' to '/home/Downloads/vosk-api/nodejs/lib/linux-x86_64/libvosk.so'

**Comment by sidor926** [comment]
Hey- I see this file /home/Downloads/vosk-api/nodejs/lib/linux-x86_64/libvosk.so where it should be- gives the same error! Thoughts?

**Comment by arthurcohen** [comment]
sorry about my ignorance, but what is missing to provide more architectures on the node package? I can see that we have all the artifacts on the releases page, and it would simply be adding those files to the package, alongside an architecture check, like `os.arch() == 'arm64'`. 

Does this sounds over-optimistic or makes sense? Can I get your input @nshmyrev?

**Comment by nshmyrev** [comment]
> it would simply be adding those files to the package, alongside an architecture check, like os.arch() == 'arm64'.

Yes, it just needs testing. I don't have arm64 to test but you are welcome to submit a patch.

**Comment by zhdan88vadim** [comment]
> Then, I just copy the libvosk.so from python module vosk from '/usr/local/lib/python3.9/dist-packages/vosk/libvosk.so'' to '/home/Downloads/vosk-api/nodejs/lib/linux-x86_64/libvosk.so'

This solution worked for me too. I was able to run on Raspberry PI 3.
Thank you!
